src/Enhavo/Bundle/SidebarBundle/Controller/SidebarController.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: philippsester
+ * Date: 24.05.19
+ * Time: 21:01
+ */
+
+namespace Enhavo\Bundle\SidebarBundle\Controller;
+
+use Enhavo\Bundle\AppBundle\Controller\ResourceController;
+
+class SidebarController extends ResourceController
+{
+    public function showResourceAction($contentDocument)
```

Comment:
Sidebars are not rendered by a controller, also this function is never called so you can remove this function